site2/docs/io-cdc-debezium.md

```diff
@@ -220,6 +221,7 @@ You can use one of the following methods to create a configuration file.
         database.dbname: "postgres"
         database.server.name: "dbserver1"
         schema.whitelist: "inventory"
+         "plugin.name": "pgoutput",
```

Comment:
Please fix the indent, and it's better to remove`""` of `plugin.name` to keep the same code style.